platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/EmbeddedSolrFactory.java

```diff
@@ -31,33 +32,45 @@
 import org.apache.solr.core.SolrCore;
 import org.apache.solr.core.SolrResourceLoader;
 import org.apache.solr.schema.IndexSchema;
+import org.codice.solr.factory.SolrClientFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.xml.sax.InputSource;
 import org.xml.sax.SAXException;
 
-public class EmbeddedSolrFactory extends SolrClientFactory {
+import com.google.common.util.concurrent.Futures;
 
-    public static final String DEFAULT_EMBEDDED_CORE_NAME = "embedded";
+public class EmbeddedSolrFactory implements SolrClientFactory {
 
-    public static final String IMMEMORY_SOLRCONFIG_XML = "solrconfig-inmemory.xml";
+    protected static final Logger LOGGER = LoggerFactory.getLogger(EmbeddedSolrFactory.class);
```

Comment:
I think the HTTP Solr factory used to extend the embedded factory.  I missed this protected after splitting them apart.